demo-snippet.html

```diff
@@ -55,29 +48,32 @@
 
 <dom-module id="demo-snippet">
   <template>
-    <style is="custom-style" include="prism-theme-default"></style>
```

Comment:
This breaks syntax highlighting in shadow dom. Do you plan to add this back or supply a replacement?